app/src/main/java/com/sun/music61/data/source/remote/TracksRemoteDataSource.java

```diff
@@ -15,4 +35,58 @@ public static TracksRemoteDataSource getInstance() {
 
     private TracksRemoteDataSource() {
     }
+
+    @Override
+    public List<Track> getBanners() {
+        List<Track> banners = new ArrayList<>();
+        HashMap<String, String> response;
+        String tags = "song";
+        String limit = "5";
+        String offset = "1";
```

Comment:
chuyển thành hằng số và tạo ở đầu phần thân class e. tương tự với các biến trên
```suggestion
       private static final String OFFSET_NUMBER = "1";
```